web/frontend/src/pages/form/GroupedResult.tsx

```diff
@@ -124,9 +124,15 @@ const GroupedResult: FC<GroupedResultProps> = ({ rankResult, selectResult, textR
           const select = element as SelectQuestion;
 
           if (selectResult.has(id)) {
-            res = countSelectResult(selectResult.get(id)).resultsInPercent.map((percent, index) => {
-              return { Candidate: select.Choices[index], Percentage: `${percent}%` };
-            });
+            res = countSelectResult(selectResult.get(id))
+              .map(([, totalCount], index) => {
```

Comment:
The linter doesn't complain? I thought you'd have to write `.map(([_, totalCount]` here.

Reply:
no, the linter is quite happy like that and actually made me change it to this :woman_shrugging: